### PR TITLE
Fdo container template

### PR DESCRIPTION
**Pull request by deviantony**
Minimum container configuration to run the FDO Owner Service container:

```
docker run --rm \
-p 8042:8042 -p 8043:8043 \
-v fdo_owner:/home/fdo/target \
-e owner_to0_rv_blob=http://IP:8042?ipaddress=IP\
-e owner_api_user=portainer \
-e owner_api_password=portainer \
-e owner_to0_scheduling_interval=60s \
-e owner_to2_port=8042 \
-e owner_to0_scheduling_enabled="true" \
-e owner_protocol_scheme=https \
-e owner_https_port=8043 \
-e owner_keystore=./owner_keystore.p12 \
-e owner_keystore_password=JPmYuPdSYOG8U3a \
-e owner_ssl_keystore=certs/ssl.p12 \
-e owner_ssl_keystore-password=Sw2cqdcc1IM \
-e ssl_truststore=certs/truststore \
-e ssl_truststore_password=Sw2cqdcc1IM \
-e log4j_configuration_file=log4j2.xml \
-e catalina_home=./target/tomcat \
-e owner_database_connection_url=jdbc:h2:tcp://localhost:8051/./target/data/ops \
-e owner_database_username=sa \
-e owner_database_password= \
-e owner_database_port=8051 \
-e owner_svi_string=./serviceinfo/sample-svi.csv \
-e owner_svi_values=./serviceinfo/sample-values \
portainer/pri-fidoiot:owner-master
```

The application will start and output the following warnings:

```
20:56:08.202 [WARN ] Could not load property: ondie_cache
20:56:08.203 [WARN ] Could not load property: epid_online_url
20:56:08.203 [WARN ] Could not load property: epid_test_mode
20:56:08.210 [WARN ] Could not load property: owner_pub_key_path
20:56:08.211 [WARN ] Could not load property: owner2_pub_key_path
```

Not sure whether this is important or not.

Which leaves us with the (currently) optional env vars:
```
-e ondie_cache=file:///home/fdo/ondie_cache/ \
-e ondie_autoupdate="false" \
-e ondie_zip_artifact=https://tsci.intel.com/content/csme.zip \
-e ondie_check_revocations="false" \
-e fido_ssl_mode=TEST \
-e owner_pub_key_path=owner_pub_keys.pem \
-e owner2_pub_key_path=owner2_pub_keys.pem \
```

### CODE REVIEW

File: templates-2.0.json [hunk 1]
```diff
@@ -1,6 +1,81 @@
...
+					"default":"60s",
+					"preset":true
+				},
+				{
+					"name":"owner_https_port",
+					"default":"8043",
+					"preset":true
+				},				
+				{
+					"name":"owner_keystore",
```
Comment (deviantony) [inline templates-2.0.json:48]: I assume that this was generated at the Docker image build time following these instructions:
https://github.com/secure-device-onboard/pri-fidoiot/blob/2bac84514d02c89583585d3dfaa305681625c82a/component-samples/demo/owner/README.md#inserting-keys-into-owner-keystore

Comment (deviantony) [reply templates-2.0.json:48]: Confirmed with @andres-portainer

File: templates-2.0.json [hunk 2]
```diff
@@ -1,6 +1,81 @@
...
+					"name":"owner_api_user",
+					"label":"API user"
+				},
+				{
+					"name":"owner_api_password",
+					"label":"API password"
+				},
+				{
+					"name":"owner_to0_scheduling_interval",
+					"default":"60s",
```
Comment (andres-portainer) [inline templates-2.0.json:39]: Is the `s` allowed?

Comment (deviantony) [reply templates-2.0.json:39]: Well that's the thing, in your template you are using `60` as the value but if I read the official documentation, it specifies that the default value for `owner_to0_scheduling_interval` is `300s`: https://github.com/secure-device-onboard/pri-fidoiot/tree/2bac84514d02c89583585d3dfaa305681625c82a/component-samples/demo/owner#configuring-the-fdo-pri-owner-sample

**Pull request closed without merging**